src/algoliasearch.helper.js

```diff
@@ -248,6 +250,55 @@ AlgoliaSearchHelper.prototype.searchOnce = function(options, cb) {
   });
 };
 
+/**
+ * Start the search for answers with the parameters set in the state.
+ * This method returns a promise.
+ * @return {AlgoliaSearchHelper}
+ * @return {promise.<FacetSearchResult>} the answer results
+ */
+AlgoliaSearchHelper.prototype.searchForAnswers = function(attributesForPrediction, queryLanguages, nbHits) {
+  var state = this.state;
+  var derivedHelper = this.derivedHelpers[0];
+  if (!derivedHelper) {
+    return Promise.resolve([]);
+  }
+  var derivedState = derivedHelper.getModifiedState(state);
+  var params = merge(
+    {
+      attributesForPrediction: attributesForPrediction,
+      nbHits: nbHits
+    },
+    omit(requestBuilder._getHitsSearchParams(derivedState), [
+      'attributesToSnippet',
+      'hitsPerPage',
+      'restrictSearchableAttributes',
+      'clickAnalytics',
+      'facets',
+      'highlightPostTag',
+      'highlightPreTag',
+      'maxValuesPerFacet',
+      'page',
+      'snippetEllipsisText',
+      'tagFilters'
```

Comment:
what's the reason for omitting these parameters?

Reply:
It was my bad putting things in a wrong object.